packages/terra-demographics-banner/src/DemographicsBanner.jsx

```diff
@@ -0,0 +1,179 @@
+/* eslint-disable global-require, import/no-dynamic-require, react/no-unused-prop-types, react/no-danger */
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+
+import './DemographicsBanner.scss';
+
+const propTypes = {
+  /**
+   * Additional Details to display in the banner.
+   */
+  additionalDetails: PropTypes.node,
+  /**
+   * Age of the person.
+   */
+  age: PropTypes.string,
+  /**
+   * Additional rows of key value pairs to add to the demographics banner. The rows will be added after the person rows.
+   */
+  applicationRows: PropTypes.array,
+  /**
+   * The persons date of birth
+   */
+  dateOfBirth: PropTypes.string,
+  /**
+   * The person's deceased date. Will display the banner as deceased if this value is provided
+   */
+  deceasedDate: PropTypes.string,
+  /**
+   * Gender of the Person
+   */
+  gender: PropTypes.string,
+  /**
+   * The persons gestational age.
+   */
+  gestationalAge: PropTypes.string,
+  /**
+   * Additional key value identifiers of a person's demographic information
+   */
+  identifiers: PropTypes.object,
+  /**
+   * Full Name of the person
+   */
+  personName: PropTypes.string,
+  /**
+   * Photo to display in the banner
+   */
+  photo: PropTypes.node,
+  /**
+   * The column layout in which to present the displays.
+   */
+  postMenstrualAge: PropTypes.string,
+  /**
+   * The persons preferred first name if they have one.
+   */
+  preferredFirstName: PropTypes.string,
+};
+
+const defaultProps = {
+  additionalDetails: null,
+  age: '--',
+  applicationRows: [],
+  dateOfBirth: 'DOB: --',
+  deceasedDate: null,
+  gender: '--',
+  gestationalAge: null,
+  identifiers: {},
+  personName: '--',
+  photo: null,
+  postMenstrualAge: null,
+  preferredFirstName: null,
+};
+
+class DemographicsBanner extends React.Component {
+  additionalApplicationRows() {
+    return this.props.applicationRows.map((row, index) => (
+      // eslint-disable-next-line react/no-array-index-key
+      <div key={`additional-row-${index}`} className="terra-DemographicsBanner--row">
+        {Object.keys(row).map(key => (
+          <span key={`additional-row-${key}-${row[key]}`}>{key}: <b>{row[key]}</b></span>
+        ))}
+      </div>
+    ));
+  }
+
+  personDetails() {
+    const elements = [
+      <span key="age"><b>{ this.props.age }</b></span>,
+      <span key="gender"><b>{ this.props.gender }</b></span>,
+      <span key="dob" dangerouslySetInnerHTML={{ __html: this.props.dateOfBirth }} />,
+    ];
+
+    if (this.props.gestationalAge) {
+      elements.push(<span key="ga" dangerouslySetInnerHTML={{ __html: this.props.gestationalAge }} />);
```

Comment:
Some of the requirements of the previous banner allow for the passing of content such as DOB: <b>{date}</b>. We need to allow the developer the ability to pass in the bold tag